cfe_internal/enterprise/federation/federation.cf

```diff
@@ -314,7 +319,11 @@ bundle agent clean_when_off
       "has_cftransport_fcontext" expression => returnszero("$(paths.semanage) fcontext -l | grep $(home)", "useshell");
 
   commands:
-    # _stdlib_path_exists_<command> and paths.<command> are defined is masterfiles/lib/paths.cf
+      # Oh, the humanity! Where for art thou databases: promises for psql!
```

Comment:
The problem here is that if you put a feeder to target_state off you will get errors as no changes are made to the database, so need to guard with is_superhub here.

Reply:
![as-you-wish](https://user-images.githubusercontent.com/202896/126793069-e99a53b9-f80a-4ac2-832a-a1e3dc8e02d0.gif)

Reply:
My comment wasn't right. Any hub 3.15.x and forward has the two cfsettings database tables. The issue here really is only do this if a hub and the version is 3.15 or greater. Currently https://tracker.mender.io/browse/ENT-8294 is a problem with triple upgrade CI test because an upgrade from 3.12.7 to 3.15.5 will break because 3.12.7 hubs don't have these tables in cfsettings.